src/Compilers/CSharp/Portable/Binder/Binder_WithExpression.cs

```diff
@@ -64,39 +63,22 @@ private BoundExpression BindWithExpression(WithExpressionSyntax syntax, Diagnost
                 }
 
                 lookupResult.Clear();
-                // PROTOTYPE: discarding use-site diagnostics
```

Comment:
It looks like there is also a PROTOTYPE comment on line 53, do we still consider it to be relevant or is overriding/hiding Clone an unsupported scenario?

Reply:
I think it's half-relevant in that I haven't thought about hiding. But it should handle overriding -- at least, my understanding of LookupMembersInType is that it will stop looking through base types when it finds a candidate in a derived type.